locustfile.py

```diff
@@ -60,3 +64,17 @@ def idemia_enrollment(self):
             headers={"Authorization": self.bearer_token},
             name="/idemia/enrollment/uuid",
         )
+
+
+class SKTestUser(FastHttpUser):
+    wait_time = between(1,1)
+    def on_start(self):
+        """ on_start is called when a Locust start before any task is scheduled """
+        pass
+    def on_stop(self):
+        """ on_stop is called when the TaskSet is stopping """
+        pass
+    @task(1)
+    def hello_world(self):
+        self.client.post(f"https://{SK_API_ROUTE}/v1/company/wdK3fH48XuoXzvZyeNJEYFA9i8K72BZg/flows/IU1iDIvviIth5jiYmNvgsS43Kg29RxyB/start",
+        data={},auth=None,headers={"x-sk-api-key":SK_API_KEY})
```

Comment:
This won't pass `Black` formatting. Make sure you have installed the pre-commit hooks defined in the `.pre-commit-config.yaml` file and check out the [black editor integrations](https://black.readthedocs.io/en/stable/integrations/editors.html)